test/test_pyosmium_get_changes.py

```diff
@@ -105,6 +105,7 @@ def test_init_from_seq_file(self):
             fd = open(fname, 'r')
             content = fd.read()
             assert_equals('454', content)
+            fd.close()
```

Comment:
This (and the close() above) should go into the finally section, so that they are also executed on test failure.

Reply:
I'm not sure it will work in all cases: if `assert_equals(0, self.main('-f', fname))` fails, then fd won't be opened, so fd.close() will fail.

I can align with the code in `test_init_to_file()` if you prefer.

Reply:
Yes, the try should go after the open to avoid that.